### DISCUSSION TITLE
LOD graphics feature to increase render distance and reduce lag with no external power

### DISCUSSION THREAD

**Discussion by Atlyz** [QuestCraftPlusPlus/QuestCraft#351] [Ideas]
LOD stands for level of detail. This is a feature that minecraft doesn't have. What minecraft naturally does is load every single pixel even if it is 100s of blocks away, we do not end up seeing this level of detail, all this is doing is making our worlds more laggy for no reason. The normal solution to this is to make things far away not as detailed as things close up by decreasing the amount of polygons in far away distances, this is how games such as COD, Fortnite, GTA, etc are able to load and run these huge maps without any performance issues. If we can implement this into QuestCraft lag would decrease, render distances could be placed higher, and overall player enjoyment would skyrocket due to the incredible immersion that being able to see long distances would do. I think if QuestCraft really wants to perform better, implementing this feature is a no brainer. Thank you for your time.

**Comment by mikeyridingabikey** [comment]
I think this Would be a great idea, this issue being i don't believe Minecraft used polygons, and if that's the case little to nothing can be done, there is a resolution slider if you want to turn the level of detail down in the Pojav launcher settings

**Comment by Rickeyrabbit** [comment]
thers a mod for 1.18.2 that does this, but it doesnt work on questcraft. we could have this feature easily if support for that 1 mod could be added
